app/.eslintrc

```diff
@@ -0,0 +1,58 @@
+{
+  "env": {
+    "amd": true,
+    "es6": true,
+    "jasmine": true,
+    "jquery": true,
+    "node": true,
+    "mocha": true,
+    "browser": true,
+    "builtin": true
+  },
```

Comment:
It does. By default too, if I recall correctly. So we'd probably only need to override whatever `env`s we need and any special rule overrides.

For example, since we're in "./app/", I think we should be setting `node: false` and `es6: false`.

nit: Plus maybe convert this to YAML too and A-Z sort the `env`s above.